birdbox/birdbox/settings/base.py

```diff
@@ -12,13 +12,23 @@
 from os.path import abspath
 from pathlib import Path
 
+from django.utils.log import DEFAULT_LOGGING
+
 import dj_database_url
 import sentry_sdk
-from everett.manager import ConfigManager
+from everett.manager import ConfigManager, ListOf
 from sentry_sdk.integrations.django import DjangoIntegration
 from sentry_sdk.integrations.logging import ignore_logger
 from wagtail.embeds.oembed_providers import vimeo, youtube
 
+try:
+    from dotenv import load_dotenv
+
+    load_dotenv()  # take environment variables from .env.
+except ImportError:
+    # dotenv is not available - e.g. in production mode
+    pass
```

Comment:
This is possibly a bit over-cautious of me, but dotenv is only installed as dev deps to avoid a situation where a .env file added to a prod build would inject perhaps unexpected vars into the env

Thoughts welcome - unnecessary caution and complexity?

Reply:
Everett has https://everett.readthedocs.io/en/latest/environments.html#env-files-configenvfileenv

Curious if we can configure dev mode to add that to the configuration but leave it out in production deploys?

Reply:
Looking the docs, I think we'd have to add in a `DEV=True|False`-style thing like we do in bedrock to then decide which env vars to pull in. As such, I'm not sure there's a clear win either way?

Reply:
you could set and env var in the Dockerfile if you just want to know in which environment you're running. This is essentially the same thing but with looking for a dep right? I think rob is just saying that Everett already has the dotenv functionality.